pkg/skuba/actions/node/upgrade/apply.go

```diff
@@ -57,6 +58,11 @@ func Apply(client clientset.Interface, target *deployments.Target) error {
 	fmt.Printf("Latest Kubernetes version: %s\n", latestVersion)
 	fmt.Printf("Current Node version: %s\n", nodeVersionInfoUpdate.Current.KubeletVersion.String())
 	fmt.Println()
+	target.IsSUSEOS()
+	if strings.Contains(target.Cache.OsRelease["VERSION_ID"], "15.1") {
```

Comment:
Nit: I would probably use a constant for 15.1, so that the comparison is clearer. Not 100% necessary.

Reply:
So guess what! Change of plans! The node info is gathered by kubelet, so we can assume it's always up to date.

If you can just compare the node info to SP1, that would be good.

``nodeVersionInfoUpdate.Current.Node.Status.NodeInfo.OSImage`` contains the string ``SUSE Linux Enterprise Server 15 SP1`` for example.

Reply:
Well, that was wrong. I think we need to load it live, as the deployer might not have rebooted after SP2 migration yet.